lisa/environment.py

```diff
@@ -206,9 +226,11 @@ def create(
         id_: int,
```

Comment:
It looks we can remove the `create` method, to create new instance from `__init__` directly.

Reply:
originally, there may be different code path to create env, but for new, the logic is pretty simple. Let's remove the class method `create`.

Reply:
My first thought was to remove this method as well as we discussed offline. However, it seems like nodes are being initialized in different ways. For example
https://github.com/microsoft/lisa/blob/5c72dabe3151d869eb15b827758f3c9bc959fe5b/lisa/environment.py#L240-L249

https://github.com/microsoft/lisa/blob/5c72dabe3151d869eb15b827758f3c9bc959fe5b/lisa/sut_orchestrator/azure/tests/test_prepare.py#L312-L318

The code in test_prepare.py isn't calling the code in `create`. Was `create` method written to make testing easier?

Reply:
The test code looks creating the default Azure extension schema, so it should be ok to without the create method. It needs create runbook, and then pass it in.